src/libraries/System.Memory/src/System/Buffers/SequenceReader.cs

```diff
@@ -49,6 +49,11 @@ public SequenceReader(ReadOnlySequence<T> sequence)
         /// </summary>
         public readonly ReadOnlySequence<T> Sequence { get; }
 
+        /// <summary>
+        /// The unread portion of the <see cref="Sequence"/>.
```

Comment:
nit: Following existing docs pattern.
```suggestion
        /// Gets the unread portion of the <see cref="Sequence"/>.
```

Reply:
nit: Also, we haven't consistently been adding the `value` tag to properties. But we should, so they show up in the docs.

Would you mind adding one, particularly because `UnreadSpan` has it. You can borrow it from there.

Similar to:
https://github.com/dotnet/runtime/blob/3b8f1c943548bd2612794b42c715f8a53b7bd3a7/src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonNodeOptions.cs#L48-L50

Reply:
cc @mairaw, @carlossanlop